### PR TITLE
fix(playground): handle null maxTokens when opening trace in Playground

### LINKED ISSUE #1354: Open in Playground fails when maxTokens is null in trace data
## Bug Description

When using the "Open in Playground" feature with a trace from an OpenAI call, the operation fails with a Zod validation error.

## Error Message

```
Error loading span data into prompt studio
[
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "null",
    "path": ["version", "configData", "llm", "maxTokens"],
    "message": "Expected number, received null"
  }
]
```

## Root Cause

The Zod schema for `configData.llm.maxTokens` expects a `number`, but the trace data contains `null`. This can happen when:
- The OpenAI call didn't specify `max_tokens`
- The SDK captured the value as `null` rather than `undefined`

## Expected Behavior

The "Open in Playground" feature should handle `null` values for `maxTokens` gracefully, either by:
- Making the field optional/nullable in the schema (`.nullable()` or `.optional()`)
- Transforming `null` to a default value or `undefined`

## Steps to Reproduce

1. Make an OpenAI API call without specifying `max_tokens`
2. View the trace in LangWatch
3. Click "Open in Playground" on the span
4. Observe the validation error

## Acceptance Criteria

- [ ] Users can open traces in Playground even when `maxTokens` is `null`
- [ ] The schema handles nullable LLM config fields appropriately

### PR DESCRIPTION
**Pull request by drewdrewthis**
## Summary

- Fix Zod validation error when opening traces in Playground where `maxTokens` is `null` (e.g., OpenAI calls without explicit `max_tokens`)
- Add `?? undefined` null-to-undefined coercion for `maxTokens`, matching the existing `temperature` pattern
- Add unit tests covering null values, present values, and default model fallback

Closes #1354

## Test plan

- [x] Unit tests pass (`pnpm test:unit src/prompts/prompt-playground/hooks/__tests__/useLoadSpanIntoPromptPlayground.unit.test.ts`)
- [x] Typecheck passes (no new errors)
- [ ] Manual: Open a trace with null maxTokens in Playground — no validation error

🤖 Generated with [Claude Code](https://claude.com/claude-code)

# Related Issue

- Resolve #1354

### CODE REVIEW

File: specs/prompts/open-trace-in-playground.feature [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    When I build the prompt form values from the trace
+    Then the form values are created successfully
+    And maxTokens is set to 1024
+    And temperature is set to 0.7
+
+  @unit
+  Scenario: Trace has no model specified
+    Given the trace has no model specified
+    When I build the prompt form values from the trace
+    Then the form values use the default model
```
[the remark this answers is not in the document]
Comment (drewdrewthis) [reply specs/prompts/open-trace-in-playground.feature]: Fixed in 5c85326. Rewrote all scenarios to use user-visible outcomes (e.g., "Playground loads without validation errors", "max tokens is left unset") instead of internal field semantics.
